pkg/types/baremetal/validation/platform.go

```diff
@@ -85,5 +85,21 @@ func ValidatePlatform(p *baremetal.Platform, n *types.Networking, fldPath *field
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("bootstrapHostIP"), p.BootstrapProvisioningIP, err.Error()))
 	}
 
+	if p.ProvisioningInterface == "" {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("provisioningInterface"), p.ProvisioningInterface, "Baremetal provisioning interface unset."))
+	}
+
+	if p.ProvisioningNetworkCIDR == "" {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("provisioningNetworkCIDR"), p.ProvisioningNetworkCIDR, "Baremetal provisioning network CIDR not set."))
+	}
+
+	if p.ProvisioningIP == "" {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("provisioningIP"), p.ProvisioningIP, "Baremetal provisioning IP must be set"))
+	}
+
+	if p.ProvisioningDHCPRange == "" {
```

Comment:
What's the format of this? Something like "172.22.0.1-172.22.0.10"? For validation, it might be easier if it's two separate fields with a start, and end.